plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/testing/ResourceHudiTablesInitializer.java

```diff
@@ -46,11 +46,13 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static com.google.common.base.Verify.verify;
```

Comment:
The main method exists soon due to try-with-resource. Please remove it. You can refer to other query runners. e.g. PostgreSqlQueryRunner